cor_surrogates/Makefile

```diff
@@ -1,35 +1,59 @@
 ## all           : clean saved results and get results to populate report
-all: clean run_cvsl_varsets createRDAfiles_fromSLobjects tables_figures
+all: clean run_cvsl_varsets_d29 createRDAfiles_fromSLobjects_d29 tables_figures_d29 \
+	run_cvsl_varsets_d57 createRDAfiles_fromSLobjects_d57 tables_figures_d57 \
+	run_cvsl_varsets_Both createRDAfiles_fromSLobjects_Both tables_figures_Both 
 
 ## clean         : delete all SL results
 clean:
 	rm -f output/*
 	rm -f figs/*
 
-## run_cvsl_varsets   : run CV-SL on vaccine arm and each variable set
-run_cvsl_varsets: 
-	Rscript code/run_cvsl_varsets.R 1 && \
-	Rscript code/run_cvsl_varsets.R 2 && \
-	Rscript code/run_cvsl_varsets.R 3 && \
-	Rscript code/run_cvsl_varsets.R 4 && \
-	Rscript code/run_cvsl_varsets.R 5 && \
-	Rscript code/run_cvsl_varsets.R 6 && \
-	Rscript code/run_cvsl_varsets.R 7 && \
-	Rscript code/run_cvsl_varsets.R 8 && \
-	Rscript code/run_cvsl_varsets.R 9 && \
-	Rscript code/run_cvsl_varsets.R 10 
-	#sbatch code/submit_cluster_job.sh
+######################################################################
+## DAY 29 MARKERS ANALYSES
+run_cvsl_varsets_d29:
```

Comment:
Have you tried running this code? I worry that it spins up 10 `Rscript` instances at once, which may cause some difficulties on a system. The previous syntax ensured that one analysis would only be run after the previous analysis was complete.

Reply:
Just pinging @brborate about this.